webpack.config.js

```diff
@@ -35,6 +35,9 @@ const { BundleAnalyzerPlugin } = require( 'webpack-bundle-analyzer' );
 const CreateFileWebpack = require( 'create-file-webpack' );
 const ManifestPlugin = require( 'webpack-manifest-plugin' );
 const features = require( './feature-flags.json' );
+const formattedFeaturesToPHPArray = Array.isArray( features )
+	? features.map( ( feature ) => `'${ feature }'` ).join( ',' )
+	: '';
```

Comment:
We don't need the array check here because it's our own import which isn't dynamic so we can rely on it always being an array. If it wasn't something is probably wrong and it's probably better for it to fail here 👍